scripts/extras/update_task.apt

```diff
@@ -179,10 +177,10 @@ update_task() {
 upgrade_check() {
     # Find the upgradable list of fledge packages
     UPGRADABLE_LIST="sudo apt list --upgradable | grep ^fledge"
-    write_log "" "$0" "debug" "Executing $UPGRADABLE_LIST ..." "logonly"
+    # write_log "" "$0" "debug" "Executing $UPGRADABLE_LIST ..." "logonly"
     UPGRADE_CMD_OUT=$(eval $UPGRADABLE_LIST)
     UPGRADE_CMD_STATUS="$?"
-    write_log "" "$0" "debug" "Upgrade check result [$UPGRADE_CMD_OUT], retcode $UPGRADE_CMD_STATUS" "all" "pretty"
+    # write_log "" "$0" "debug" "Upgrade check result [$UPGRADE_CMD_OUT], retcode $UPGRADE_CMD_STATUS" "all" "pretty"
```

Comment:
Remove the commented lines if not needed

Reply:
intention was kept for debugging in commented way. So that in future we can enable the line and see what the issue is if anything happened on the user's system.